src/app.js

```diff
@@ -153,3 +158,10 @@ function renderVersion() {
 function reportTopLevelError(msg, url, lineNo, columnNo, error) {
   handleError(error);
 }
+
+function setThemeInfo() {
+  const preferAltTheme = JSON.parse(localStorage.getItem('preferAltTheme'));
+  const otherName = (darkMediaQuery.matches ? !preferAltTheme : preferAltTheme) ? 'light' : 'dark';
```

Comment:
|darkMediaQuery.matches|preferAltTheme|Name of "other theme" used in text|
|--|--|--|
|true|true|dark|
|true|false or undefined|light|
|false|true|light|
|false|false or undefined|dark|

Just writing this out for my own benefit and future reference.

Reply:
Thanks for the note. I am willing to say that this might not be ideal as an xor type of setup, given that some folks might have a preference for dark mode for this app in particular but run their desktop against the day/night cycle. 
But given that this is a fairly niche app, this seemed like a "good enough" solution.
A more intensive app might benefit from a more thoughtful override system.

Reply:
Yes, as a guy without access to dark mode, I have no idea what people in general would want, though we _do_ know what the two actual users of the app want.

Reply:
Why you gotta call me out like that.

Reply:
_Clicks "Resolve conversation"_